gpkit/fmincon/simpleflight.py

```diff
@@ -0,0 +1,62 @@
+"Minimizes airplane drag for a simple drag and structure model."
```

Comment:
Oh, yeah, that can be annoying. I think t_examples adds that dir to PYTHONPATH, but I'd also be fine with adding that `__init__` file to examples